common/statistics/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+# Copyright 2024 - Nym Technologies SA <contact@nymtech.net>
+# SPDX-License-Identifier: Apache-2.0
+
+[package]
+name = "nym-statistics-common"
+version = "1.1.1"
```

Comment:
Are we publishing this crate? Thinking about the version. Simplest is probably to just stick to 0.1.0

Reply:
Fixed in adf198f6b8696b963643253dd0ee3aa6e745f13c